File: C program/for table.c
#include<stdio.h>
void main()
{
	int num;
	int counter;
	int result=0;
	printf("Enter a number:");
	scanf("%d",&num);
	for(counter=1;counter<=10;counter++)
	{
		result=num*counter;
		printf("%dx%d=%d\n",num,counter,result);
	}
}

//Algorithm
//1)start
//2)declare num,counter, result=0
//3)accept num
//4)print num
//5)for counter=1;counter<=10;counter++
//6)result=num*counter
//7)print num*counter=result
//8)stop

File: Csharp_programs/for_table.cs
//4- wap to accept a no. and print table of that number

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace for_table
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int num, cnt = 1, result = 0;
            Console.WriteLine("Enter Num: ");
            num=Convert.ToInt32(Console.ReadLine());
            for(cnt=1; cnt<=10; cnt++)
            {
                result = num * cnt;
                Console.WriteLine("{0} * {1} = {2} ",num,cnt,result);
            }
            Console.ReadKey();
        }
    }
}
